app/src/main/java/eu/faircode/email/ServiceSynchronize.java

```diff
@@ -687,7 +687,7 @@ public void run() {
                         String body = Helper.readText(EntityMessage.getFile(this, message.id));
                         StringBuilder sb = new StringBuilder();
                         if (!TextUtils.isEmpty(message.subject))
-                            sb.append(message.subject).append("<br>");
+                            sb.append("<b>").append(message.subject).append("</b>").append("<br>");
```

Comment:
The formatting of the subject was deliberately kept the same expanded and not expanded, so there is a smooth transition when expanding/collapsing.

Reply:
Hmm...I thinks it is difficult to distinguish between subject and preview without highlighting it.

Reply:
How about italic as something in between? The notification title is already bold and the difference between expanded and collapse will be smaller.

Reply:
I think it is a better idea to show the text italic, just like in the message view.